server/src/js/parent-helper.js

```diff
@@ -17,10 +17,28 @@ window.onmessage = function(e) {
     if (loaded) {
       doResize();
     }
-  } else if (e.data.scrollX) {
-    let frameOffset = document.getElementById("frame").getBoundingClientRect().top;
-    let toolbarHeight = document.getElementById("toolbar").clientHeight;
-    window.scrollTo(0, frameOffset + e.data.scrollY - toolbarHeight);
+  } else if (e.data.clipWidth) {
+    window.scrollTo(0, 0);
```

Comment:
What's this for?  Note later you'll need to take into account window.scrollY for some things like the rect top